### ISSUE TITLE
Default note

### ISSUE THREAD

**Issue by knausb** [grunwaldlab/quiltr#37]
The default note (notes.Rmd) appears to be just a yaml header.  A 'skeleton' type page, similar to what is provided by Rstudio which includes some examples of links, bolds, etc. , might be nice.

**Comment by grunwald** [comment]
Agreed. It would be good to also include basic markup syntax as demo.

**Comment by zachary-foster** [comment]
Sounds like a good idea to me. I will look into it. Thanks!

On Mon, May 11, 2015 at 11:05 AM, Niklaus Grunwald <notifications@github.com

> wrote:
> 
> Agreed. It would be good to also include basic markup syntax as demo.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/grunwaldlab/labtools/issues/37#issuecomment-101001120
> .

## 

Zachary S.L. Foster, PhD Student
Oregon State University
USDA Agricultural Research Service
Horticultural Crops Research Lab
Corvallis, Oregon, USA
http://grunwaldlab.cgrb.oregonstate.edu/
